axon-framework/events/event-processors/README.md

```diff
@@ -253,12 +308,213 @@ public interface ErrorHandler {
+        this.eventProcessingConfiguration = eventProcessingConfiguration;
+    }
+
+    public void repairAllEvents() {
+        Optional<Iterable<Iterable<DeadLetter<? extends EventMessage<?>>>>> deadLetters = eventProcessingConfiguration.deadLetterQueue(
+                PROCESSING_GROUP).map(SequencedDeadLetterQueue::deadLetters);
+
+        deadLetters.ifPresent(sequences -> {
+            // Iterate over all sequences
+            for (Iterable<DeadLetter<? extends EventMessage<?>>> sequence : sequences) {
+                // Iterate over all dead-letters belonging to the same sequence
+                for (DeadLetter<? extends EventMessage<?>> deadLetterInSequence : sequence) {
+                    eventProcessingConfiguration.sequencedDeadLetterProcessor(PROCESSING_GROUP).ifPresent(
+                            eventMessageSequencedDeadLetterProcessor -> {
+                                eventMessageSequencedDeadLetterProcessor.process(
+                                        deadLetter -> deadLetter.message().getIdentifier()
+                                                                .equals(deadLetterInSequence.message()
+                                                                                            .getIdentifier()));
+                            });
+                }
+            }
+        });
+    }
+}
+```
+
+{% endtab %}
+{% endtabs %}
+
+You can implement a custom dead-letter policy to exclude some events from the dead-letter queue, these events will be
+skipped. This policy is not only called for initial failures but also when dead-lettered events are processed
+unsuccessfully again.
+
+```java
+@Configuration
+public class CustomDeadLetterPolicy {
+
+    @Autowired
+    public void configure(EventProcessingConfigurer configurer) {
+        configurer.registerDeadLetterPolicy(PROCESSING_GROUP, configuration ->
+                (letter, cause) -> {
+                    if (cause instanceof NullPointerException) {
+                        // It's pointless..
+                        return Decisions.doNotEnqueue();
+                    }
+                    final int retries = (int) letter.diagnostics().getOrDefault("retries", -1);
+                    if (letter.message().getPayload() instanceof ErrorEvent) {
+                        // Important, always retry
+                        return Decisions.enqueue(cause);
+                    }
+                    if (retries < 10) {
+                        // Let's continue and increase retries!
+                        return Decisions.enqueue(cause,
+                                                 deadLetter -> deadLetter.diagnostics().and("retries", retries + 1));
+                    }
+                    // Exhausted retries
+                    return Decisions.doNotEnqueue();
+                });
+    }
+}
+
+```
+
+One important note, when implementing event handlers, make them idempotent. With the dead-letter queue this is a hard requirement.
```

Comment:
Awesome, we definitely need this paragraph indeed.
I think it would be good if we had this as a "notice section".

The notion for that is like how you'd do references.
The `> **Ordering Considerations*` bit on line 155 is a sample of this.

Furthermore, I'd like this notice to come earlier within the DLQ section.
Perhaps even directly after the introduction of this section.
That way, readers know there handlers should be idempotent "before" they've already read everything else.